src/Appwrite/Platform/Tasks/Retry.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Utopia\CLI\Console;
+use Utopia\Platform\Action;
+use Utopia\Queue\Client;
+use Utopia\Queue\Connection;
+use Utopia\Validator\Text;
+
+class Retry extends Action
+{
+    public static function getName(): string
+    {
+        return 'retry';
+    }
+
+
+    public function __construct()
+    {
+        $this
+            ->desc('Retry Queue')
```

Comment:
lets have a better description "Retry failed jobs from a specific queue identified by the name parameter"